meetings/2023-07-26.md

```diff
@@ -0,0 +1,176 @@
+
+
+### Add W3C Solid CG Charter
+URL: https://github.com/solid/process/pull/323
+
+#### Raw data
+* SC: As of now on the PR:
+* SC: 18 participants.
+* SC: 12 reviews and approvals, 0 requested changes, 1 comment (editorial).
+* SC: 3 thumbs-up and 2 thumbs-downs emoji reactions on the original comment. Various exciting emojis throughout the thread.
+* SC: 15 commits integrating feedback from the reviews, meetings, and elsewhere.
+* SC: 127 conversations.
+* SC: Some brief/General discussions in mailing list, Solid forum, Matrix chat.
+* SC: Thank you for all feedback. If you haven't yet reviewed, please review. This charter is by us for us!
+* SC: Some open discussions as follows:
+
+#### License
+* SC: See thread https://github.com/solid/process/pull/323#pullrequestreview-1512907580 discussing MIT and W3C CLA/FSA licenses.
+* SC: Current state: pending W3C guidance.
+* SC: Is it useful to get a hold of W3C legal?
+* PAC: I'll ping a few people and point them to the PR.
+* SC: Could we agree to leave the section on licensing as is until we hear back? My understanding is that us using MIT right now does not prevent us from creating snapshots of CG drafts and finals. My suggestion is to wait until we hear back before accepting the charter. 
+* eP: I'd like to propose to copy and paste the discussion into a separate issue and PAC can point to that separate issue. 
+
+ACTION: eP to create the issue. done: https://github.com/solid/process/issues/327
+
+
+#### UI layer metadata
+URL: https://github.com/solid/process/pull/323/files#r1270051384
+
+* SC: Raised by RB.
+* SC: I believe it refers to whether descriptions of UI can be referred to in vocabularies, etc. I mentioned SolidOS' UI vocab. 
+* SC: Can others review the questions? Is this generally covered by the Scope? Should it be explicitly listed?
+* eP: In terms of what will be published by CG, would that be a note? 
+* SC: My interpretation would be vocab and possibly shapes. Solid CG is the only working on it, SolidOS is interested in it.. so we should acknowledge. 
+* WT: I think that everything that will be done on UI level should/could rely on the work done in application interoperability and the panes and shapes are important but should be gathered under the work item of interop. 
+* SC: We have data shapes as an item under scope. Should we revise that item to emphasize UI? 
+* WT: Could change wording to something more general, maybe mention relationship between data/display thereof. 
+* eP: I think it doesn't seem to be out of scope. New work item describes workflow describes how new working items can be proposed for the CG. I think this should just go through this workflow.
+* SC: We don't say vocabularies but we could mention alongside data shapes.
+* WT: I think that it's in scope but need to mention specifically shapes/vocab when the point is interoperability. Maybe mentioning as examples.
```

Comment:
The way this was recorded seemed to be interpretable as exactly the opposite of what I meant.

```suggestion
* WT: I think that it's certainly in scope but only mentioning shapes/vocab is too specific when the point is interoperability. Maybe mention those as examples.
```